tests/acceptance/test_bootimg.py

```diff
@@ -17,16 +17,13 @@
 
 import pytest
 
-from common import (
-    build_image,
-    latest_build_artifact,
-)
+from utils.common import build_image, latest_build_artifact
```

Comment:
Mmm. Are you using a different `black` version? I am pretty sure I formatted this previously with the tool

Reply:
I'll check.